Automattic/newspack-plugin
# Adds meta pixel settings section

### All Submissions:


### Changes proposed in this Pull Request:



Closes #1554 .

Relevant discussion: https://wp.me/pamTN9-3ZQ

Adds an option to the Engagement > Social settings page to allow users to configure Meta (Facebook) pixel.

### How to test the changes in this Pull Request:

1. Go to Newspack > Engagement > Social
2. Activate "Meta Pixel"
3. Add a pixel ID
4. Save
5. Test invalid formats
6. Test enabling and disabling the option
7. Check the copy and the link
8. Visit the site and confirm the pixel was added to the page. You can use a browser extension to help you with that if you want: https://href.li/?https://chrome.google.com/webstore/detail/facebook-pixel-helper/fdgfkebogiimcoedlicjlajpkdmockpc?hl=en
9. Visit the site with AMP enabled and confirm the pixel is also there
10. I've only tested this with random IDs. It would be awesome if someone could test it with a real pixel ID in a real site

### Other information:
